client/src/app/submission/submission.component.html

```diff
@@ -22,58 +22,85 @@ <h2 class="vf-text vf-text-heading--2">
          href="{{submissionEnvelope?._links?.self?.href}}">
         <mat-icon matTooltip="See Ingest API JSON">open_in_new</mat-icon>
       </a>
+      <button mat-icon-button [matMenuTriggerFor]="menu">
+        <mat-icon>more_vert</mat-icon>
+      </button>
+      <mat-menu #menu="matMenu" xPosition="before">
+        <button mat-menu-item (click)="selectedIndex=4">
+          <mat-icon>upload_file</mat-icon>
+          <span>Import a spreadsheet</span>
+        </button>
+      </mat-menu>
       <app-submission-state [state]="submissionState">{{submissionState}}</app-submission-state>
     </div>
+
+  </div>
+
+  <div fxLayout="row" fxLayoutAlign="space-between center">
+    <div>
+      <div class="vf-text-body vf-text-body--2">{{projectTitle}}</div>
+      <div class="vf-text-body vf-text-body--3">{{getContributors(project)}}</div>
+    </div>
+    <div></div>
   </div>
+
   <div>
-    <div class="vf-text-body vf-text-body--2">{{projectTitle}}</div>
-    <div class="vf-text-body vf-text-body--3">{{getContributors(project)}}</div>
     <div *ngIf="validationSummary && validationSummary.totalInvalid > 0">
-      <p>Your validation returned <span class="error-text">{{validationSummary.totalInvalid}} error(s)</span>. Review and fix them below.</p>
+      <p>Your validation returned <span class="error-text">{{validationSummary.totalInvalid}} error(s)</span>. Review
+        and fix them below.</p>
 
       <ul id="validation-error-types">
         <li *ngIf="validationSummary.invalidBiomaterials > 0">
           Biomaterials:
-          <a href="javascript: void(0)" (click)="navigateToTab(0, { dataSource: biomaterialsDataSource, filterState: 'Invalid'})">
+          <a href="javascript: void(0)"
+             (click)="navigateToTab(0, { dataSource: biomaterialsDataSource, filterState: 'Invalid'})">
```

Comment:
The tab numbers as literals is not a good practice. Have you considered using constants for them?

Reply:
Done.